File: documentation/user/en/query/filtering/examples/references/reference-having-exact-id.cs
EvitaResponse<ISealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	FilterBy(
        		ReferenceHaving(
        			"brand",
        			EntityPrimaryKeyInSet(66465)
        		)
        	),
        	Require(
        		EntityFetch(
        			AttributeContent("code"),
        			ReferenceContentWithAttributes("brand")
        		)
        	)
        )
	)
);

File: documentation/user/en/query/filtering/examples/references/reference-having-exact-id.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				filterBy(
					referenceHaving(
						"brand",
						entityPrimaryKeyInSet(66465)
					)
				),
				require(
					entityFetch(
						attributeContent("code"),
						referenceContentWithAttributes("brand")
					)
				)
			)
		);
	}
);